src/objective-c/examples/BazelBuildSamples/ios-sample/Podfile

```diff
@@ -0,0 +1,31 @@
+platform :ios, '8.0'
```

Comment:
If it's a bazel sample, maybe we should avoid having a `Podfile` in the directory to confuse users.

Reply:
@muxi My intent was to show how both could work  (was about to add a readme to clarify that), also to avoid creating one project for each OS x build system. That would be too many sample dummy projects. I'm not sure if all of them are needed, though. What do you think?

Reply:
@muxi each OS x build system = iOS-Cocoapods, iOS-Bazel, tvOS-Cocoapods, tvOS-Bazel, watchOS-Cocoapods, watchOS-Bazel. And potentially macOS as well?